m/mullvad/stone.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : mullvad
+version     : 2025.8
+release     : 1
+homepage    : https://mullvad.net
+upstreams   :
+    - git|https://github.com/mullvad/mullvadvpn-app.git : 94fee68b5a83fe1df0cbc99a3d303fd7ef15519f
+summary     : The Mullvad VPN client app for desktop
+description : |
+    The Mullvad VPN client app for desktop
+license     : GPL-3.0-or-later
+networking  : true
+builddeps   :
+    - pkgconfig(alsa)
+    - pkgconfig(gbm)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(protobuf)
+    - gcc
+    - git
+    - golang
+    - nodejs
+    - rust
+rundeps    :
+    - libayatana-appindicator
+setup       : |
+    %patch %(pkgdir)/0001-pack-to-folder.patch
+    %patch %(pkgdir)/0002-update-build.sh.patch
+build       : |
+    # Due to the limitation of the go build command, it's not possible to specify that the tool should use ar or llvm-ar. This seems to be hardcoded, so the symbolic link appears to be the best solution.
+    ln -s /usr/bin/llvm-ar /usr/bin/ar
+    ./build.sh --optimize
+install     : |
+    %install_dir %(installroot)%(datadir)/%(name)
```

Comment:
While solus uses `/usr/share/` for Electron apps we use `/usr/lib/` on Aeryn. Electron apps contain compiled architecture-specific code and datadir is more for arch-independent stuff.